src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -363,23 +364,40 @@ export class NotebookModel extends Disposable implements INotebookModel {
 						}
 					});
 				}
+				// Get Notebook URI Params & adjust to string
```

Comment:
Not sure what the "adjust to string" means

Reply:
// Modify Notebook URI Params format from URI query to string space delimited format